core/templates/dev/head/domain/utilities/UrlInterpolationServiceSpec.js

```diff
@@ -205,4 +205,32 @@ describe('URL Interpolation Service', function() {
     })).toThrow(new Error(
       'Parameters passed into interpolateUrl must be strings.'));
   });
+
+  it('should interpolate correct imagePath', function() {
+    expect(uis.getStaticImageUrl('/test_url/')).toBe(
+      GLOBALS.ASSET_DIR_PREFIX + '/assets/images' + '/test_url/');
```

Comment:
Have the expected value be as simple as possible, i.e.

    GLOBALS.ASSET_DIR_PREFIX + '/assets/images/test_url/'